modules/core/02-client/exported/exported.go

```diff
@@ -0,0 +1,68 @@
+package exported
+
+import (
+	"context"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	upgradetypes "github.com/cosmos/cosmos-sdk/x/upgrade/types"
+	clienttypes "github.com/cosmos/ibc-go/v6/modules/core/02-client/types"
+	"github.com/cosmos/ibc-go/v6/modules/core/exported"
+)
+
+// ClientKeeper defines the expected interface of the light client used for IBC
```

Comment:
Maybe we could add some short documentation explaining why this interface exists? That is to extend the ClientKeeper implementation (the ibc module client keeper implementation should not have its logic modified, only extended)

Reply:
Maybe we could add additional documentation [here](https://github.com/cosmos/ibc-go/blob/main/docs/ibc/integration.md), via a section "Extending the client keeper"